packages/verification-client/src/Client.js

```diff
@@ -96,6 +120,201 @@ export default class Client {
     })
   }
 
+  /**
+   * Verifies a file hash based on claims
+   * @param {string} hash
+   * @return {FileVerification}
+   */
+  async verifyFileClaimBased (hash) {
+    let result=await this.resolveAndValidateFileClaim(hash)
```

Comment:
Agree, correct spacing would be with spaces before and after the =, but as this will be completly overhauled anyway, it doesn't matter that much at the moment in my opinion.

Reply:
ok :) i resolve